README.md

```diff
@@ -1,23 +1,34 @@
-# react-native-point-sdk
+# Point SDK React Native
 
-## Add private pod specs repo to your cocoa pods local installation
+Access comprehensive health and fitness user data collected from across multiple wearable devices.
 
-Our Swift PointSDK pod is published under this private specs repo, so you'll need to add it to your cocoa pods repo list first
+## Overview
 
-`$ pod repo add point-specs git@github.com:agencyenterprise/point-specs.git`
+Point SDK provides an easy, “plug-and-play” way for you to get access to the full-range of user health metrics powered by Point.
 
-## Getting started
+With Point SDK, you can:
 
-`$ npm install react-native-point-sdk --save`
+- Collect user health and fitness data through Apple HealthKit and have it processed by Point.
+- Decide the granularity of health and fitness data you would like to use from the wearable devices, depending on your app needs.
+- Retrieve digested health metrics in a normalized, consistent data format across all devices
 
-### Mostly automatic installation
+Point SDK provides a high-level interface for you to setup some listeners on your app, which will be watching for new wearables data coming from Apple HealthKit and proceed to process this data asynchronously.
 
-`$ react-native link react-native-point-sdk`
+Point is continually deriving new health metrics, health score updates, personalized health insights, and workout recommendations based on the wearables data, and you can retrieve those through our SDK at any time to deliver a custom experience to your users.
+
+## Getting your credentials
+
+Once you've completed your onboarding process with us, you'll receive an api key and your first clientId and clientSecret to start using our SDK! For more information about out onboarding process, please contact [tech@areyouonpoint.co](mailto:tech@areyouonpoint.co)
+
+## Registering and authenticating your users
 
-## Usage
+See [Authenticating Users](https://point-landing-pages.vercel.app/docs/authenticating-users) to learn how to link your user accounts with Point Database.
 
-```javascript
-import PointSdk from "react-native-point-sdk";
-// TODO: What to do with the module?
-PointSdk;
-```
+## Adding to your project
+
+See [Getting Started](https://point-landing-pages.vercel.app/docs/rn-getting-started) to learn how to configure you project and start using the SDK.
+
+### Mostly automatic installation
+
+`$ npm install react-native-point-sdk --save`
+`$ react-native link react-native-point-sdk`
```

Comment:
we don't need to `link` since RN already does that automatically depending on its version.